SemanticSearchV2.md

```diff
@@ -185,22 +183,39 @@ The document response is divided into two parts. First is the parsed_input node
 			- job_titles (for type: job_title, skill, company, keyword)
 			- job_level (for type: job_title, skill, company, keyword)
 			- education_level (for type:school)
-	- versions (map): relationship strings mapped to version strings
-
-
+	- location: possible fields are address, city, country, region, state, zip.
+	
+	 *(The following fields are only available when requesting a job document.)*
+	- company_geographies (list): contains geography objects.
+	- contract_type (string): whether the contract is permanent, temporary, internship, etc.
+	- education_level (string): minimum education level required by the job posting.
+	- employment_level (string): whether the job is full-time, part-time, etc.
+	- experience_level: possible fields are name, level, min_years, max_years.
```

Comment:
Shouldn't this be something more like "possible values are..."? It's not the case that there are multiple fields.

Reply:
I doubt that "possible values" is better. These are fields of experience_level object.
for each of these fields, it makes sense to say like "possible values of min_years field could be integer between 1 and 10."